packages/components/src/card/card-media/component.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import { contextConnect } from '../../ui/context';
+import { View } from '../../view';
+import { useCardMedia } from './hook';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').MediaProps, 'div'>} props
+ * @param {import('react').Ref<any>} forwardedRef
+ */
+function CardMedia( props, forwardedRef ) {
+	const mediaProps = useCardMedia( props );
+
+	// Should it add the `components-card__media` className ?
```

Comment:
The original implementation explicitly added the `components-card__media` className, is it something we should carry over to this implementation too?

Reply:
Yup